bundles/org.openhab.binding.boschshc/src/main/resources/Smart Home Controller Issuing CA.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What's the source of this certificate?

Reply:
AS stated in the commit message of the pem, the files are from here https://github.com/BoschSmartHome/bosch-shc-api-docs/tree/master/best_practice

Reply:
@fwolter Should we document the source of the certificates somewhere?

Reply:
Yes, you could add a Developer.md in the root directory of the binding with a short notice.

Reply:
I will add a ``DEVELOPERS.md``

Reply:
@GerdZanker added a DEVLOPERS.md file with some descriptions about the certificates.